sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -2554,12 +2554,16 @@
 
   /**
    * @since 4.4
+   * @deprecated since 5.5
    */
+  @Deprecated
   public static final String QUALITY_PROFILES_KEY = "quality_profiles";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)